src/Bridges/bridge_optimizer.jl

```diff
@@ -1398,25 +1350,30 @@ function MOI.supports(
     attr::MOI.AbstractConstraintAttribute,
     IndexType::Type{MOI.ConstraintIndex{F,S}},
 ) where {F,S}
-    return reduce_bridged(
-        b,
-        F,
-        S,
-        true,
-        () -> MOI.supports(b.model, attr, IndexType),
-        ok ->
-            ok && MOI.supports(
```

Comment:
@odow Maybe that needed to be `||` and we can keep your translation below and just revert the `if` and `return` to reflect that

Reply:
Hmm. Potentially. I refactored these methods because they were impenetrable, and I really didn't understand what they were trying to do.